instances/latest/terminologies/technique/multiPhotonFluorescenceMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/multiPhotonFluorescenceMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
```

Comment:
Suggestion based on ["2P microscopy"](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/technique/twoPhotonFluorescenceMicroscopy.jsonld): "Multi photon fluorescence microscopy is a fluorescence microscopy technique for living tissue which is based on the simultaneous excitation by two or more photons with longer wavelength than the emitted light."

Reply:
done